.github/workflows/test-and-publish.yaml

```
@@ -0,0 +1,97 @@
+    tags:
+      - 'v*'  # Push events to version tags v1.0, v20.15.10
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  code-quality:
+    name: Code Quality
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
```

yamllint

[warning] wrong indentation: expected 6 but found 4

(indentation)